homeassistant/components/http.py

```diff
@@ -221,28 +220,6 @@ def send(self, request, filepath):
 _GZIP_FILE_SENDER = GzipFileSender()
 
 
-class HAStaticRoute(StaticRoute):
-    """StaticRoute with support for fingerprinting."""
-
-    def __init__(self, prefix, path):
-        """Initialize a static route with gzip and cache busting support."""
-        super().__init__(None, prefix, path)
-        self._file_sender = _GZIP_FILE_SENDER
-
-    def match(self, path):
-        """Match path to filename."""
-        if not path.startswith(self._prefix):
-            return None
-
-        # Extra sauce to remove fingerprinted resource names
```

Comment:
@balloob I took another look at this today, I'm not sure how to do this - freeze stops us from adding anything after we start running. How can/should we ensure that we've added all the views before starting?

Also it appears that aiohttp_cors isn't compatible with the latest aiohttp release.